client/v2/offchain/verify.go

```diff
@@ -0,0 +1,141 @@
+package offchain
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/encoding/prototext"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	apitx "cosmossdk.io/api/cosmos/tx/v1beta1"
+	v2flags "cosmossdk.io/client/v2/internal/flags"
+	authsigning "cosmossdk.io/x/auth/signing"
```

Comment:
is there a way to avoid using x/auth/signing? x/tx should be the package we use for everything related to signing